src/settings/OwnerSettings.test.js

```diff
@@ -179,13 +177,14 @@ describe('Owner settings', () => {
   // fill out form elements
   // click the "save" button
   // wait for the "save" button to disappear
-  it.skip('Create and edit functionality', async () => {
+  it('Create and edit functionality', async () => {
```

Comment:
Here, new button click is being tested and not edit button. 
1. As per the test title, edit also needs to be tested.
2. is it possible to simulate save on new record and edit record?

Reply:
Hey, I created a separate test case to cover the edit functionality.